packages/ruleset/src/functions/precindition-header.js

```diff
@@ -0,0 +1,42 @@
+module.exports = function(operation, _opts, context) {
```

Comment:
please rename file to be `precondition-header.js`

Reply:
Oops, done!